data/Delimiter/cpp_char.yaml

```diff
@@ -0,0 +1,5 @@
+type: string
```

Comment:
remove this , and use `c_char.yaml` - they are the same delimiter definition , I believe.

If someone disagrees, they can split them into two with proof they are different.